tests/API/XCCDF/unittests/test_remediation_metadata.sh

```diff
@@ -17,16 +27,35 @@ echo "Result file = $result"
 
 $OSCAP xccdf generate fix --fix-type bash --profile 'common' --output "$result" "$input_xml" 2> $stderr
 
-grep -q "^\s*#.*Profile title on one line" "$result"
-grep -q "^\s*#\s*Profile description" "$result"
-grep -q "^\s*#\s*that spans two lines" "$result"
+case $(uname) in
+	FreeBSD)
+		grep -q "^#.*Profile title on one line" "$result"
+		grep -q "^#.*Profile description" "$result"
+		grep -q "^#.*that spans two lines" "$result"
+		;;
+	*)
+		grep -q "^\s*#.*Profile title on one line" "$result"
```

Comment:
It look like we don't need `\s*`s even in the Linux case, do we? I mean, could we just use FreeBSD version for everything?

Reply:
You are right, I've changed this and the others you pointed out.